e2e/snapshot-escape/__tests__/snapshot.test.js

```diff
@@ -8,3 +8,4 @@
 
 // prettier-ignore
 test('escape strings', () => expect('one: \\\'').toMatchSnapshot());
+test('escape strings two', () => expect('two: \'"').toMatchSnapshot());
```

Comment:
i think this line is not supposed to be committed.
i believe it's coming from our old janky snapshot tests that modifies our fixtures (that are under git and not in /tmp) and if you interrupt the test before it's done this thing will stay there :( (hence all other modifications to snapshots you had to make)

check `e2e/__tests__/snapshot.test.js`

Reply:
Ah ok - should I remove this file or that line?

Reply:
I saw other PRs failing with the same error